games/armagetronad/Portfile

```diff
@@ -0,0 +1,291 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           boost 1.0
+
+github.setup        ArmagetronAd armagetronad 0.2.9.1.0 v
+github.tarball_from releases
+extract.suffix      .tbz
```

Comment:
So it's a bzip2-compressed tarball? Then you need to specify `use_bzip2 yes` so that MacPorts uses the right extraction command. The "wrong" extraction command (the one for gzip-compressed tarballs) happens to work on OS X 10.9 and later, but earlier systems require the correct command.